modules/ossm-cr-example.adoc

```diff
@@ -3,26 +3,141 @@
 // * service_mesh/v2x/customizing-installation-ossm.adoc
 
 [id="ossm-cr-example_{context}"]
-= {ProductName} custom resources
+= Control plane parameters 
 
-[NOTE]
-====
-The `istio-system` project is used as an example throughout the {ProductShortName} documentation, but you can use other projects as necessary.
-====
+The following table lists the top-level parameters for the `ServiceMeshControlPlane` resource.
 
-A _custom resource_ allows you to extend the API in an {ProductName} project or cluster. When you deploy {ProductShortName} it creates a default `ServiceMeshControlPlane` that you can modify to change the project parameters.
+.`ServiceMeshControlPlane` resource parameters
+|===
+|Name |Description |Type
+
+|`apiVersion`
+|APIVersion defines the versioned schema of this representation of an object. Servers convert recognized schemas to the latest internal value, and may reject unrecognized values. The value for {ProductName} version 2.0 is `maistra.io/v2`.
+|The value for {ProductName} version 2.0 is `maistra.io/v2`.
```

Comment:
I think this sentence creates confusion. `maistra.io/v2` is not related to the OSSM version but rather the version of the SMCP. This has caused a lot of confusion with OSSM 2.0 where the `apiVersion` and OSSM version match, but this is not guaranteed to match. For instance, both OSSM 1.1 and 1.0 use `maistra.io/v1`